magento/magento2
# Set visibility for product's option according to current website

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#28291. Configurable shows simple products which are no longer assigned to the website

### Manual testing scenarios (*)

1. Create additional website, store, storeview "other" in backend
2. Create Configurable with Options, assign to both websites
3. Set Quantity and in stock for each option in backend
4. Check that all 3 options are there in the frontend as expected
image
5. Disable option 1 in the "main website"
6. Flush caches

### Questions or comments


### Contribution checklist (*)
